app/utils/format.js

```diff
@@ -1,19 +1,25 @@
-import moment from 'moment'
+import dayjs from 'dayjs'
+import 'dayjs/locale/ja'
+import relativeTime from 'dayjs/plugin/relativeTime'
+dayjs.extend(relativeTime)
 
 export function formatDate(date) {
-  return moment(date, 'X')
+  return dayjs
+    .unix(date)
     .locale('ja')
-    .format('L')
```

Comment:
you can use `LocalizedFormat` plugin for `L LLL LTS` support
https://github.com/iamkun/dayjs/blob/dev/docs/en/Plugin.md#localizedformat

Reply:
@xxyuk Thanks for your comment !
We will use LocalizedFormat.